ste/uploader-blobFS.go

```diff
@@ -169,22 +175,44 @@ func (u *blobFSUploader) GenerateUploadFunc(id common.ChunkID, blockIndex int32,
 	})
 }
 
+var flushThreshold int64 = (1024 * 1024 * 1024) * 20 //20GB
```

Comment:
block size might vary for each file, right? so this should perhaps not be a global variable